client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -79,6 +80,8 @@ class ContentDataProvider
   public dropMimeTypes: string[];
   public dragMimeTypes: string[];
 
+  private uriToParentMap = new Map<string, string>();
```

Comment:
I guess the cache should be cleared when [refresh](https://github.com/sassoftware/vscode-sas-extension/blob/main/client/src/components/ContentNavigator/ContentDataProvider.ts#L445)